src/database/migrations/20231213033935_addDescriptiveNameToApiRoles.ts

```diff
@@ -0,0 +1,27 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.schema.alterTable('apiRoles', (table) => {
+    table.string('externalName', 128);
+  });
+
+  await knex('apiRoles').where({ roleName: 'GENERATOR' }).update({ externalName: 'Token Creator' });
+  await knex('apiRoles').where({ roleName: 'MAPPER' }).update({ externalName: 'UID2 Creator' });
+  await knex('apiRoles').where({ roleName: 'ID_READER' }).update({ externalName: 'Decryptor' });
+  await knex('apiRoles').where({ roleName: 'SHARER' }).update({ externalName: 'Securer' });
+  await knex('apiRoles').where({ roleName: 'OPTOUT' }).update({ externalName: 'OPTOUT' });
```

Comment:
nit: perhaps the externalName should be "Opt-out"

Reply:
Sounds good! Have changed.